src/main/java/hybrid/it/internship/library/service/impl/BookServiceImpl.java

```diff
@@ -34,7 +34,7 @@
     @Transactional(readOnly = true)
     public BookDTO getById(Long id) {
         return bookMapper.toDTO(bookRepository.findById(id)
-                .orElseThrow(() -> new EntityNotFoundException(id.toString())));
+                .orElseThrow(() -> new EntityNotFoundException()));
```

Comment:
Here set the error message, that the book entity is not found and by which id was it searched for in the DB.